packages/nuxt/src/app/composables/router.ts

```diff
@@ -109,6 +109,12 @@ export interface NavigateToOptions {
   open?: OpenOptions
 }
 
+const SCRIPT_PROTOCOLS_RE = /^(data|javascript|vbscript):$/
```

Comment:
@pi0 What do you think about moving to UFO?

Reply:
Haha you read my mind was _just_ about to mention this. We could have a utility like `isSafeProtocol` over there for the check i likely want to also include this fix for h3 (not sure if redirect can be affected or not)